foundry/loaders/tf_wrapper.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+from tensorflow.keras.utils import Sequence
+
+class TensorflowSequence(Sequence):
+    """Foundry Dataset Converted to Tensorflow Format"""
+
+    def __init__(self, inputs, targets):
+        self.inputs=inputs
+        self.targets=targets
+
+    def __len__(self):
+        return len(self.inputs[0])
+
+    def __getitem__(self, idx):
+        item = {"input": [], "target": []}
+        
+        for input in self.inputs:
```

Comment:
I was poking through this code again, and am a little concerned about this part. 

We're combining all of our inputs into a single array, which will only work with the following line that combines them into a single array if they are the same shape. 

I can see this working if your different inputs are, for example, different color channels from the same image, but not if one is an image and the other is a 1D array associated with it.

I'm not yet sure what real-world datasets we want to use with this, so I don't have recommendations for what the behavior should be yet.

Could you write a line comment about what this loop is supposed to do to make it easier to tune later?

Reply:
Since it is not a numpy array, they don't need to all have the same shape.